staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -94,27 +97,26 @@ func getResourceHandler(scope RequestScope, getter getterFunc) http.HandlerFunc
 		ctx := scope.ContextFunc(req)
 		ctx = request.WithNamespace(ctx, namespace)
 
-		result, err := getter(ctx, name, req)
+		result, err := getter(ctx, name, req, trace)
 		if err != nil {
 			scope.err(err, w, req)
 			return
 		}
+		trace.Step("About to create self-link")
 		if err := setSelfLink(result, req, scope.Namer); err != nil {
 			scope.err(err, w, req)
 			return
 		}
+		trace.Step("About to write a response")
 		responsewriters.WriteObject(http.StatusOK, scope.Kind.GroupVersion(), scope.Serializer, result, w, req)
 	}
 }
 
 // GetResource returns a function that handles retrieving a single resource from a rest.Storage object.
 func GetResource(r rest.Getter, e rest.Exporter, scope RequestScope) http.HandlerFunc {
+	// For performance tracking purposes.
```

Comment:
No - it's a new line. Before it was next to "trace" declaration. Now it's useless.